### DISCUSSION TITLE
No being able to verify my account as Github Student

### DISCUSSION THREAD

**Discussion by ghost** [community/community#41898] [GitHub Education]
My issue is about how I need to prove the validity. I have tried to share the document the institution provided me anyhow. I can not print it at the moment. Is there another way I could verify my status? The institution mentioned there is a deal between GitHub and CF(Career Foundry).
Any further info would be appreciated.

**Comment by SilasRodrigues19** [comment]
@HerRA17 Hi.

Take a photo or screenshot of the institution's document and also use the institutional email. Good luck
